src/module.cpp

```diff
@@ -128,3 +137,23 @@ void Module::VerifyFunction() {
 		}
 	}
 }
+std::string Module::MapName(std::string moduleName, std::string specieName,
```

Comment:
The module should already know what it's own name is, so there is no need to supply it as an argument